keel-core-test/src/main/kotlin/com/netflix/spinnaker/keel/persistence/DeliveryConfigRepositoryTests.kt

```diff
@@ -215,6 +221,20 @@ abstract class DeliveryConfigRepositoryTests<T : DeliveryConfigRepository, R : R
             get { metadata }.isNotEmpty()
           }
       }
+
+      test("config can be rechecked") {
+        val items = repository.itemsDueForCheck(Duration.ofMinutes(2), 1)
+        val items2 = repository.itemsDueForCheck(Duration.ofMinutes(2), 1)
+        repository.triggerRecheck(deliveryConfig.application)
+        val items3 = repository.itemsDueForCheck(Duration.ofMinutes(2), 1)
```

Comment:
Consider renaming for clarity:
```suggestion
        val firstCheck = repository.itemsDueForCheck(Duration.ofMinutes(2), 1)
        val secondCheck = repository.itemsDueForCheck(Duration.ofMinutes(2), 1)
        repository.triggerRecheck(deliveryConfig.application)
        val afterRecheck = repository.itemsDueForCheck(Duration.ofMinutes(2), 1)
```